server/services/qBittorrent/clientRequestManager.ts

```diff
@@ -36,7 +36,7 @@ const EMPTY_SERVER_STATE = {
 class ClientRequestManager {
   private connectionSettings: QBittorrentConnectionSettings;
   private apiBase: string;
-  apiVersion: string | null = null;
+  apiVersion: Promise<string | undefined> = Promise.resolve(undefined);
```

Comment:
why Promise here?

Reply:
Ok I got it

Reply:
Yeah I'm not sure if there is a better way to handle it, Having it update along side the `authCookie` seems like the easiest way to do it, and it needs to be await-able so that starting / stopping doesn't resolve to undefined before the request has completed.